pytest_fixtures/component/provision_pxe.py

```diff
@@ -306,8 +314,8 @@ def provisioning_hostgroup(
         root_pass=settings.provisioning.host_root_password,
         operatingsystem=module_provisioning_rhel_content.os,
         ptable=default_partitiontable,
-        subnet=module_provisioning_sat.subnet if not settings.server.is_ipv6 else None,
-        subnet6=module_provisioning_sat.subnet if settings.server.is_ipv6 else None,
+        subnet=module_provisioning_sat.subnet if not sat_ipv6 else None,
+        subnet6=module_provisioning_sat.subnet if sat_ipv6 else None,
```

Comment:
For the dual stack provisioning follow up: `module_provisioning_sat` should expose both `subnet` and subnet6`.

Reply:
Yes, we should but that is when dualstack provisioning will be suppported and would also need `configure-sat-provisioning` workflow updates